velox/common/base/BloomFilter.h

```diff
@@ -42,18 +59,65 @@ class BloomFilter {
     bits_.resize(std::max<int32_t>(4, bits::nextPowerOfTwo(capacity) / 4));
   }
 
+  bool isSet() {
+    return bits_.size() > 0;
+  }
+
   // Adds 'value'.
-  void insert(uint64_t value) {
+  // case:
+  // InputType can be one of folly hasher support type when hashInput = false.
+  // InputType can only be uint64_t when hashInput = true.
+  template <class InputType = uint64_t, bool hashInput = true>
```

Comment:
I am thinking about removing the templating here.  Since the hash function is only applied on input value, user could do it outside this class.  What do you think?

Reply:
I agree, this is the old logical, so I keep it here. I will remove it.